tcf_website/tests/test_utils.py

```diff
@@ -70,16 +75,76 @@ def setup(obj):
         username="tcf2yay",
         email="tcf2yay@virginia.edu",
         computing_id="tcf2yay",
+        first_name="Taylor",
+        last_name="Comb",
+        graduation_year=2023
     )
     obj.user2 = User.objects.create(
         username="tcf3yay",
-        email="tcf3yay@virginia.edu",
         computing_id="tcf3yay",
+        last_name="No first name"
+    )
+    obj.user3 = User.objects.create(
+        username="bnf89798",
+        computing_id="bnf89798",
+        last_name="Bam Friedman"
+    )
+    obj.user4 = User.objects.create(
+        username="kik878",
+        computing_id="kik878",
+        last_name="Kjell Kool"
     )
 
     obj.instructor = Instructor.objects.create(
         last_name="Jefferson",
-        first_name="Tom"
+        first_name="Tom",
+        email="tjt3rea@virginia.edu"
+    )
+
+    obj.section_course = Section.objects.create(
+        course=obj.course,
+        semester=obj.semester,
+        sis_section_number=312312
+    )
+
+    obj.section_course.instructors.set(
+        Instructor.objects.filter(
+            first_name="Tom"))
```

Comment:
Did you mean to use a `QuerySet` that includes the `Tom Jefferson` instance above (i.e. `obj.instructor`) or any instructors with the first name `Tom`? If the former is the case, the following code seems more appropriate, since someone might add another `Instructor` with `first_name` `Tom`:
```suggestion
    obj.section_course.instructors.set(
        Instructor.objects.filter(
            pk=obj.instructor.pk))
```
If you do decide to change this, make sure to change this elsewhere.